msg/Status.msg

```diff
@@ -2,16 +2,21 @@
 uint16 NORMAL=0
 uint16 SETTING=1
 uint16 operating_mode
-
-# The configured area number the stop occurred in.
 uint16 area_number
 # If the laser is reporting an error or not.
-bool error_status
+bool error_state
```

Comment:
Did this field need to be renamed? I'm thinking of message compatibility between upstream and our fork.

Reply:
I renamed for what the lidar documentation is referring. But now that I think of it, I might as well just leave it as it was! Will rename this.